{{cookiecutter.repository_name}}/README.md

```diff
@@ -1,10 +1,116 @@
+<p align="left">
+<img width=15% src="https://dai.lids.mit.edu/wp-content/uploads/2018/06/Logo_DAI_highres.png" alt=“DAI-Lab” />
+<i>An open source project from Data to AI Lab at MIT.</i>
+</p>
+
 [![PyPI Shield](https://img.shields.io/pypi/v/{{ cookiecutter.package_name }}.svg)](https://pypi.python.org/pypi/{{ cookiecutter.package_name }})
 [![Travis CI Shield](https://travis-ci.org/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }}.svg?branch=master)](https://travis-ci.org/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }})
+{%- if cookiecutter.use_codecov == 'y' %}
+[![Coverage Status](https://codecov.io/gh/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }}/branch/master/graph/badge.svg)](https://codecov.io/gh/{{ cookiecutter.github_owner }}/{{ cookiecutter.repository_name }})
+{%- endif %}
+[![Downloads](https://pepy.tech/badge/{{ cookiecutter.package_name }})](https://pepy.tech/project/{{ cookiecutter.package_name }})
```

Comment:
But the opposite is also true: some of the projects will be released to PyPI, and commenting or removing the badge is as easy as uncommenting it.

The difference is that if it is uncommented since the beginning, the badge can be seen when the README is rendered, so it works as a visual tip to remind the user to remove it if it is not necessary.

Another option would be to add a question for it in the cookiecutter.json (whether there will be a PyPI release or not), but we might end up having too much stuff there.

Reply:
Actually, the same happens with the `Install` section. By default we indicate that the recommended way to install the package is by using the PyPI version, so the user needs to remove that section if that is not true.

But that looks much better than doing the opposite, and not including the section by default, as it's always much easier to remove unneeded things than to add them if they are needed.

Reply:
I see what you are saying.

Here is my thinking

| Problem | Cost| 
| -------- | ---- |
| Project not on PyPI | Installation instructions are inaccurate, download badge shows error |
| Project on PyPI | User has to remember to uncomment badge |

The costs are asymmetric. It may very well be months and months before a new project is on PyPI. I still recommend not including this by default.

How about this? We consider, "does the user intend to distribute this on PyPI very soon?" We ask this as a single prompt (`intend_to_distribute_on_pypi`) and include information related to PyPI (download count, installation instructions, etc.) if they say yes.

Reply:
I tried adding a question, but it was not self-explaining enough, and it was confusing because of the other `use_pypi_deployment_with_travis` question.
So I simply commented out the two badges and the `Install using pip` (which I renamed to `Install from PyPI`) section, adding comments indicating that the section and the badges need to be uncommented after the PyPI release.